fuel/app/config/routes.php

```diff
@@ -4,4 +4,5 @@
 	'_404_'   => 'welcome/404',    // The main 404 route
 	
 	'hello(/:name)?' => array('welcome/hello', 'name' => 'hello'),
+  'profile(/:name)?' => array('welcome/profile', 'name' => 'profile')
```

Comment:
__[code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed
* Array key not aligned correctly; expected 8 spaces (or 1 tabs) but found 2
* Array with multiple values cannot be declared on a single line

<!-- Comment by [SideCI](https://sideci.com) -->